src/modules/account/internal/account-writer.ts

```diff
@@ -0,0 +1,20 @@
+import { Account, CreateAccountParams } from '../types';
+import AccountReader from './account-reader';
+import AccountUtil from './account-util';
+import AccountRepository from './store/account-repository';
+
+export default class AccountWriter {
+  public static async createAccount(
+    params: CreateAccountParams,
+  ): Promise<Account> {
+    const AccountDB = AccountRepository.accountDb;
```

Comment:
This syntax is not desirable. As I mentioned earlier, we should be able to import `AccountDB` directly from `AccountRepository`